games-emulation/fbalpha_libretro/fbalpha_libretro-0.2.97.42.git.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="A port of Final Burn Alpha to the libretro architecture"
+DESCRIPTION="Final Burn Alpha is a multi-systems Arcade Emulator most popular for \
+emulating Neo-Geo, Capcom, Konami, and Cave games. It is developed by the Final Burn \
+team and originated from FinalBurn by Dave."
```

Comment:
```diff
-DESCRIPTION="Final Burn Alpha is a multi-systems Arcade Emulator most popular for \
+DESCRIPTION="Final Burn Alpha is a multi-systems Arcade Emulator most popular \
-emulating Neo-Geo, Capcom, Konami, and Cave games. It is developed by the Final Burn \
+for emulating Neo-Geo, Capcom, Konami, and Cave games. It is developed by the \
-team and originated from FinalBurn by Dave."
+Final Burn team and originated from FinalBurn by Dave."
```
(We like to have lines of up to 80 chars (including the backslash) but no more.)

Reply:
My mistake, I thought this only applied to the text within the description. Will fix that.